src/main/java/org/cryptomator/ui/vaultoptions/MountOptionsController.java

```diff
@@ -106,6 +156,21 @@ public void initialize() {
 			mountPointToggleGroup.selectToggle(mountPointDirBtn);
 		}
 		mountPointToggleGroup.selectedToggleProperty().addListener(this::selectedToggleChanged);
+
+		vaultVolumeTypeChoiceBox.getItems().add(null);
+		vaultVolumeTypeChoiceBox.getItems().addAll(mountProviders);
+		vaultVolumeTypeChoiceBox.setConverter(new MountServiceConverter());
+		boolean autoSelected = vaultSettings.mountService.get() == null;
+		vaultVolumeTypeChoiceBox.getSelectionModel().select(autoSelected ? null : selectedMountService.getValue());
```

Comment:
instead of creating the local variable `autoSelected`, you can use the `defaultMountService` binding.